docs/querying/aggregations.md

```diff
@@ -422,6 +389,121 @@ It is not possible to determine a priori how well this aggregator will behave fo
 
 For these reasons, we have deprecated this aggregator and recommend using the DataSketches Quantiles aggregator instead for new and existing use cases, although we will continue to support Approximate Histogram for backwards compatibility.
 
+
+## Expression aggregators
+
+### Expression aggregator
+
+Query time only aggregator that can aggregate results using [Druid expressions](./math-expr.md) functions to facilitate building custom functions.
+
+| property | description | required |
```

Comment:
The rows of the **Required** column should say Yes or No instead of true or false.
For example:
| `type` | Must be `expression`. | Yes |